app/src/main/res/layout/navigation_bar.xml

```diff
@@ -40,6 +40,13 @@
                 android:layout_marginStart="10dp"
                 android:src="@drawable/ic_icon_home" />
 
+            <org.mozilla.vrbrowser.ui.UIButton
```

Comment:
this is for testing right?

Reply:
Well, we want this to land. It only shows when the pref is on.
We will think of a better UI later I guess.